src/main/java/frc/robot/subsystems/Input.java

```diff
@@ -16,7 +16,7 @@ private Input() {
 
   private static final Joystick mRotJoystick = new Joystick(JoystickConstants.ROT_JOYSTICK_PORT);
   private static final Joystick mTransJoystick = new Joystick(JoystickConstants.TRANS_JOY_PORT);
-  private static final XboxController mXboxController = new XboxController(JoystickConstants.XBOX_CONTROLLER_PORT);
+  public static final XboxController mXboxController = new XboxController(JoystickConstants.XBOX_CONTROLLER_PORT);
```

Comment:
Don't do this, make getter methods

Reply:
I realized that I didn't have to make new method. I made the controller a private variable now.